bokeh/models/glyphs.py

```diff
@@ -1159,32 +1159,41 @@ class Rect(XYGlyph, LineGlyph, FillGlyph, HatchGlyph):
     """)
 
 class Scatter(Marker):
-    ''' Render arbitrary markers according a specification.
+    ''' Render scatter markers selected from a predefined list of designs.
 
-    The Scatter can draw any built-in marker type. It can be configured
-    to draw the same marker for all values by specifying the name of a
-    marker, e.g.
+The Scatter marker can draw any of Bokeh's built-in marker types: ``asterisk``,
```

Comment:
@bryevdv This might be a good place to not only list all possible marker types but to also include your proposed script that displays examples of each marker type.

However, I was wondering if there are any better locations to have an exhaustive list and demonstration of all marker types. Can you think of any better location (something along the lines of [MarkerTypes ](https://docs.bokeh.org/en/dev/docs/reference/core/enums.html?highlight=markertype#bokeh.core.enums.MarkerType) that would be referenced from several other places)?

Reply:
@tcmetzger I think it would be good to show the plot here. The script itself if a bit gorpy and I don't think we should display the script itself. A single code snippet (or small script) demonstrating passing a marker type to `Scatter` seems sufficient. 

If the plot can be displayed in the enum docs, that seems fine too. But there may be issues with including a plot from enum docstrings (will just have to try and see)

Reply:
Commit 7275477 adds the plot to glyphs.py. The docstring already includes two short code snippets for selecting a specific marker or referencing a list of markers through a data column.